android/src/main/kotlin/com/squareup/square_mobile_payments_sdk/mappers/PaymentMapper.kt

```diff
@@ -62,6 +61,14 @@ class PaymentMapper {
                 if(paymentParameters.get("autocomplete") != null) {
                     builder.autocomplete(paymentParameters.get("autocomplete") as? Boolean ?: false)
                 }
+
+                if(paymentParameters.get("autocomplete") != null) {
+                    builder.autocomplete(paymentParameters.get("autocomplete") as? Boolean ?: false)
+                }
+
+                if(paymentParameters.get("idempotencyKey") != null) {
+                    builder.paymentAttemptId(paymentParameters.get("idempotencyKey") as? String)
+                }
```

Comment:
We also need a new payment Parameter `paymentAttemptId` to match the [updated api](https://developer.squareup.com/docs/changelog/mobile-logs/2025-05-29)
iOS doesn't have the `paymentAttemptID` yet, afaik, so it would only be a wiring for an Android and no-op for iOS